### PR TITLE
Add a new example PWM setup with Symbiflow build

### PR DESCRIPTION
**Pull request by wsipak**
The example setup uses PWM IP Core generated with LiteX and can be built for ZYNQ7000 FPGAs using Symbiflow toolchain.

### CODE REVIEW

File: docs/start.rst [hunk 1]
```diff
@@ -81,6 +81,30 @@ If you don't create any template a default template bundled with Topwrap will be
...
+In order to generate the top module, run::
+
+    make generate
+
+In order to generate bitstream, add Vivado to your path and run::
+
+    make
+
+
+The FPGA design utilizes an AXI-mapped PWM IP Core. You can access its registers starting from address ``0x4000000`` (that's the base address of ``AXI_GP0`` on ZYNQ).
```
Comment (tgorochowik) [inline docs/start.rst]: Please keep one sentence per line

File: examples/pwm/sources/zynq_video_board.xdc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+set_property IOSTANDARD LVCMOS33 [get_ports pwm]
+#set_property PACKAGE_PIN N22 [get_ports pwm]
```
Comment (tgorochowik) [inline examples/pwm/sources/zynq_video_board.xdc]: Should this be removed?

File: docs/start.rst (hunk 1 above)
Comment (wsipak) [reply docs/start.rst]: Ok, done

File: examples/pwm/sources/zynq_video_board.xdc (hunk 2 above)
Comment (wsipak) [reply examples/pwm/sources/zynq_video_board.xdc]: Yes, the unneeded line was deleted and the filename is `zynq.xdc` now

**Review by tgorochowik — APPROVED** [review]

**Pull request merged**